packages/eslint-config-airbnb-base/rules/best-practices.js

```diff
@@ -164,9 +164,9 @@ module.exports = {
     'no-octal-escape': 'error',
 
     // disallow reassignment of function parameters
-    // disallow parameter object manipulation
+    // disallow parameter object manipulation except req, request, and $scope
     // rule: http://eslint.org/docs/rules/no-param-reassign.html
-    'no-param-reassign': ['error', { props: true }],
+    'no-param-reassign': ['error', { props: true, ignorePropertyModificationsFor: ['req', 'request', '$scope'] }],
```

Comment:
let's include `acc` for `reduce` accumulators, and `res` and `response` for express responses, and `e` for `e.returnValue =`.